v1.20/talos/PRODUCT.yaml

```diff
@@ -0,0 +1,9 @@
+vendor: Talos Systems
+name: Talos
+version: v0.6.1
```

Comment:
@rsmitty this should be 0.9.0 I guess

Reply:
argh, totally right. fixed in amended commit. thank you!